src/main/resources/application.yml

```diff
@@ -43,8 +43,14 @@ queueconfig:
   uac-qid-created-exchange: uac-qid-created-exchange
   invalid-address-inbound-queue: case.invalidAddressQueue
   undelivered-mail-queue: case.undeliveredMailQueue
+  outbound-field-exchange: action-outbound-exchange
+  ccs-property-listed-queue: case.ccsPropertyListedQueue
```

Comment:
binding should be configurable here

Reply:
not sure what you mean @NickGrantONS - what's different to ones above?

Reply:
sorry, I meant _topic_ not binding

Reply:
Would be nice if you could call it "topic" instead of "binding" but it's cosmetic. My bad.